[REQUEST] Update README example

**Is your feature request related to a problem? Please describe.**
It bothers me that the example given on the README doesn't compiles straight away with -Wall -Werror & -Wextra
Even more because with the recommended Makefile does indeed include this flags, so my proposal would be to fix it.

**Describe the solution you'd like**
The issues are as follows:
![Screenshot from 2024-01-23 13-34-09](https://github.com/codam-coding-college/MLX42/assets/28810331/9c191f72-381f-45e0-9ade-c7302308591f)

1. `srcs/main.c:25:31: error: comparison of integer expressions of different signedness: ‘int32_t’ {aka ‘int’} and ‘uint32_t’ {aka ‘unsigned int’} [-Werror=sign-compare]` (for both the loops)
2. `srcs/main.c:23:25: error: unused parameter ‘param’ [-Werror=unused-parameter]`
3. `srcs/main.c:58:22: error: unused parameter ‘argc’ [-Werror=unused-parameter]`
4. `srcs/main.c:58:40: error: unused parameter ‘argv’ [-Werror=unused-parameter]`

Solutions:

1. use `uint32_t` instead of `int32_t`
2. `(void) param;`
3. & 4. `int main(void) {`


If you allow me, I can submit a PR with this changes :D

Cheers, and keep it up, this library is Gold! :trophy:

--- a/README.md
+++ b/README.md
@@ -268,9 +268,10 @@ int32_t ft_pixel(int32_t r, int32_t g, int32_t b, int32_t a)
 
 void ft_randomize(void* param)
 {
-	for (int32_t i = 0; i < image->width; ++i)
+	(void)param;
+	for (uint32_t i = 0; i < image->width; ++i)
 	{
-		for (int32_t y = 0; y < image->height; ++y)
+		for (uint32_t y = 0; y < image->height; ++y)
 		{
 			uint32_t color = ft_pixel(
 				rand() % 0xFF, // R
@@ -301,7 +302,7 @@ void ft_hook(void* param)
 
 // -----------------------------------------------------------------------------
 
-int32_t main(int32_t argc, const char* argv[])
+int32_t main(void)
 {
 	mlx_t* mlx;
 

